src/argilla/client/feedback.py

```diff
@@ -0,0 +1,454 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from __future__ import annotations
+
+import sys
+import warnings
+from typing import TYPE_CHECKING, Any, Dict, Iterator, List, Optional, Tuple, Union
+from uuid import UUID
+
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
+from pydantic import (
+    BaseModel,
+    Field,
+    StrictInt,
+    StrictStr,
+    ValidationError,
+    create_model,
+    validator,
+)
+from tqdm import tqdm
+
+import argilla as rg
+
+if TYPE_CHECKING:
+    from argilla.client.api import Argilla
+    from argilla.client.sdk.v1.datasets.models import (
+        FeedbackDatasetModel,
+        FeedbackFieldModel,
+        FeedbackItemModel,
+        FeedbackQuestionModel,
+    )
+
+FETCHING_BATCH_SIZE = 250
+
+
+class ValueSchema(BaseModel):
+    value: Union[StrictStr, StrictInt]
+
+
+class ResponseSchema(BaseModel):
+    values: Dict[str, ValueSchema]
+    status: Literal["submitted", "missing", "discarded"]
+
+
+class OfflineFeedbackResponse(ResponseSchema):
+    user_id: Optional[UUID] = None
+
+    @validator("user_id", always=True)
+    def user_id_must_have_value(cls, v):
```

Comment:
This won't work for now. The current endpoint accepts responses only for connected user. I think we can remove the user_id attribute for now in the `OfflineFeedbackResponse` class